balance/weighting_methods/ipw.py

```diff
@@ -618,7 +646,7 @@ def ipw(
         model_weights,
     )
 
-    using_default_logistic = sklearn_model is None
+    using_default_logistic = custom_model is None
 
     if using_default_logistic:
```

Comment:
no need to keep this as a variable. Just use 
custom_model is None
And add as a comment # using_default_logistic